Brick[] blue_bricks
Brick[] red_bricks
